File: Application.Entity/Models/Core/BusinessAreaTypeRelationship.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Application.Data.Models.Core
{
    public class BusinessAreaTypeRelationship : BaseEntity
    {
        public int CustomerId { get; set; }
        public int BusinessAreaId { get; set; }
        public bool BusinessAreaType1 { get; set; }
        public bool BusinessAreaType2 { get; set; }
        public bool BusinessAreaType3 { get; set; }
        public bool IsActive { get; set; }
        
    }
}

File: Application.API/ClientApp/src/app/models/business-area-type-relationship.ts
export interface BusinessAreaTypeRelationship{
    id: number;
    customerId: number;
    customerName: string;
    businessAreaId: number;
    businessAreaName: string;
    businessAreaType1: boolean;
    businessAreaType2: boolean;
    businessAreaType3: boolean;
    isActive: boolean;
    dateCreated: Date;
    createdBy: string;
    dateModified: Date;
    modifiedBy: string;
}